### ISSUE TITLE
Download links are broken

### ISSUE THREAD

**Issue by jawshooah** [ruaridht/Spotify-Menubar#2]
The `lifeupnorth.co.uk` domain seems to be dead. Both download links from that domain (`http://lifeupnorth.co.uk/files/Spotify_Menubar.zip` and `http://lifeupnorth.co.uk/files/SIMBL%20Plugin%20(Spotify).zip`) are broken.

**Comment by ruaridht** [comment]
@jawshooah Apologies for the delay, I have been busy over the Christmas holidays and New Year. Thank you for letting me know, I think I've fixed the links. However, as you mentioned in caskroom/homebrew-cask/issues/8406 I have not updated the app for a long time and do not plan to update the app or the repository in the future.

I removed the old webpage (lifeupnorth.co.uk) and instead reverted to ruaridht.github.io. I tried to make sure I updated all links, apologies for this. The boneyard seems like a reasonable home for the current state of the app and repo.
